event/src/middleware.ts

```diff
@@ -0,0 +1,49 @@
+import { NextResponse } from 'next/server'
+import type { NextRequest } from 'next/server'
+
+import * as jose from 'jose';
+
+
+export async function middleware(request: NextRequest) {
+    let cookie = request.cookies.get('accessToken')
+    const key = new TextEncoder().encode("encoderKEY")
```

Comment:
For security, `"encoderKey"` should be a long randomly generated string stored in the environment variable.
```suggestion
    let skey : string = process.env.SECRETKEY!;
    const key = new TextEncoder().encode(skey)
```

Reply:
changed to :
```
const key = new TextEncoder().encode(skey);
```

Reply:
With skey refering to an ENV file.